.github/workflows/data-product-example-2-poc.yaml

```diff
@@ -0,0 +1,54 @@
+---
+name: POC - zip and upload _example_2
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "data-products/_example_2/**"
+      - ".github/workflows/data-product-example-2-poc.yaml"
+
+jobs:
+  test:
+    uses: ./.github/workflows/data-product-example-2-test.yaml
+
+  upload:
+    name: Zip and upload
+    runs-on: ubuntu-latest
+    needs: test
+    permissions:
+      id-token: write # This is required for requesting the JWT
+      contents: read # This is required for actions/checkout
+    defaults:
+      run:
+        working-directory: data-products/_example_2
+    strategy:
+      max-parallel: 4
+      matrix:
+        python-version: [3.10.10]
+    steps:
+      - uses: actions/checkout@24cb9080177205b6e8c946b17badbe402adc938f # v3.4.0
+        with:
+          fetch-depth: 2
```

Comment:
Is this required?

Reply:
fetch-depth: 1 -pulls the latest which is ok. We dont need to pull the entire history

Reply:
# Number of commits to fetch. 0 indicates all history for all branches and tags.
    # Default: 1
    fetch-depth: ''.

Reply:
So you dont need it